gamestonk_terminal/options/op_controller.py

```diff
@@ -139,8 +139,13 @@ def call_exp(self, other_args: List[str]):
 def menu(ticker: str):
     """Options info Menu."""
 
-    op_controller = OptionsController(ticker)
-    op_controller.call_help(None)
+    try:
+        op_controller = OptionsController(ticker)
+        op_controller.call_help(None)
+    except IndexError:
+        print("No options found for " + ticker)
+        process_input = False
+        return process_input
```

Comment:
Exactly!

Reply:
I'd probably also say to pull latest branch, and then add the commit on top of it. Because this was branched out earlier on, and it appears as if it contains commits that are already merged

Reply:
Okay. I think I got it right on #309. Also how would I change should_print_help to false when exiting the options menu to prevent it from printing the menu again?